### PR TITLE
Fix `show-associated-branch-prs-on-fork` classes

### PR DESCRIPTION
**Pull request by yakov116**
## Test URLs
https://github.com/talknsmile/TestR/branches

## Screenshot
image

### CODE REVIEW

**Review by fregante — CHANGES_REQUESTED** [review]
??? —open is not a standalone class

**Comment by yakov116** [comment]
> ??? —open is not a standalone class

Recheck the diff now 💇

**Comment by fregante** [comment]
Might as well use the entire class for both. 

Also no comma

File: source/features/show-associated-branch-prs-on-fork.tsx [hunk 1]
```diff
@@ -58,10 +58,11 @@ const getPullRequestsAssociatedWithBranch = cache.function(async (): Promise<Rec
 	cacheKey: () => 'associatedBranchPullRequests:' + getRepo()!.nameWithOwner
 });
 
+// TODO: Replace this with `State--${prInfo.state.toLowerCase()}`
```
Comment (fregante) [inline source/features/show-associated-branch-prs-on-fork.tsx:61]: If there’s no date it means it’s up for grabs immediately. Add a GHE mention with issue number of thats what you’re waiting for

Comment (yakov116) [reply source/features/show-associated-branch-prs-on-fork.tsx:61]:
```suggestion
// TODO: Replace this with `State--${prInfo.state.toLowerCase()}` GHE #4202
```

**Review by fregante — APPROVED** [review]
If it works for you

**Pull request merged**